include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -129,20 +157,43 @@ class JointMoteus: public JointBase
               can_id_( config.can_id),
               can_bus_( config.can_bus),
               moteus_kp_(config.moteus_kp),
-              moteus_kd_(config.moteus_kd){}
+              moteus_kd_(config.moteus_kd),
+              temperature_(config.temperature),
+              q_current_(config.q_current),
+              d_current_(config.d_current),
+              voltage_(config.voltage) {}
         
         /**
-         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves mode
+         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves registers
```

Comment:
I would change this to state values 
```suggestion
         * @brief Update the joint of the moteus. Converts rot/s to rad/s and saves 
         other state values
```